packages/react-jsx-babel-plugin/__tests__/__snapshots__/transform-jsx-to-react-jsx-test.js.snap

```diff
@@ -0,0 +1,374 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`transform react to jsx React.fragment to set keys and source 1`] = `
+"var _jsxFileName = \\"\\";
+var x = React.jsxDEV(React.Fragment, {}, \\"foo\\", true, {
+  fileName: _jsxFileName,
+  lineNumber: 1
+}, this);"
+`;
+
+exports[`transform react to jsx adds appropriate new lines when using spread attribute 1`] = `
+"React.jsx(Component, Object.assign({}, props, {
+  sound: \\"moo\\"
+}));"
+`;
+
+exports[`transform react to jsx arrow functions 1`] = `
+"var foo = function () {
+  var _this = this;
+
+  return function () {
+    return React.jsx(_this, {});
+  };
+};
+
+var bar = function () {
+  var _this2 = this;
+
+  return function () {
+    return React.jsx(_this2.foo, {});
+  };
+};"
+`;
+
+exports[`transform react to jsx assignment 1`] = `
+"var div = React.jsx(Component, Object.assign({}, props, {
+  foo: \\"bar\\"
+}));"
+`;
+
+exports[`transform react to jsx concatenates adjacent string literals 1`] = `
+"var x = React.jsx(\\"div\\", {
+  children: [\\"foo\\", \\"bar\\", \\"baz\\", React.jsx(\\"div\\", {
+    children: \\"buz bang\\"
+  }), \\"qux\\", null, \\"quack\\"]
+});"
+`;
+
+exports[`transform react to jsx fragment with no children 1`] = `"var x = React.jsx(React.Fragment, {});"`;
+
+exports[`transform react to jsx fragments 1`] = `
+"var x = React.jsx(React.Fragment, {
+  children: React.jsx(\\"div\\", {})
+});"
+`;
+
+exports[`transform react to jsx fragments in dev mode (no key and source) 1`] = `
+"var _jsxFileName = \\"\\";
+var x = React.jsxDEV(React.Fragment, {
+  children: React.jsxDEV(\\"div\\", {}, undefined, true, {
+    fileName: _jsxFileName,
+    lineNumber: 1
+  }, this)
+}, undefined, false);"
+`;
+
+exports[`transform react to jsx fragments to set keys 1`] = `"var x = React.jsx(React.Fragment, {}, \\"foo\\");"`;
+
+exports[`transform react to jsx nonStatic children 1`] = `
+"var _jsxFileName = \\"\\";
+var x = React.jsxDEV(\\"div\\", {
+  children: [React.jsxDEV(\\"span\\", {}, '0', true, {
+    fileName: _jsxFileName,
+    lineNumber: 3
+  }, this), React.jsxDEV(\\"span\\", {}, '1', true, {
+    fileName: _jsxFileName,
+    lineNumber: 3
+  }, this)]
+}, undefined, false, {
+  fileName: _jsxFileName,
+  lineNumber: 2
+}, this);"
+`;
+
+exports[`transform react to jsx properly handles keys 1`] = `
+"var x = React.jsx(\\"div\\", {
+  children: [React.jsx(\\"div\\", {}, \\"1\\"), React.jsx(\\"div\\", {
+    meow: \\"wolf\\"
+  }, \\"2\\"), React.jsx(\\"div\\", {}, \\"3\\")]
+});"
```

Comment:
Sorry I missed this - static callers should actually use [`jsxs`, not `jsx`](https://github.com/reactjs/rfcs/blob/createlement-rfc/text/0000-create-element-changes.md#always-pass-children-as-props).

```js
var x = React.jsxs("div", {
  children: [React.jsx("div", {}, "1"), React.jsx("div", {
    meow: "wolf"
  }, "2"), React.jsx("div", {}, "3")]
});
```